mongo/database.go

```diff
@@ -165,12 +165,15 @@ func (db *Database) processRunCommand(ctx context.Context, cmd interface{},
 		readSelect = makePinnedSelector(sess, readSelect)
 	}
 
-	createCmdFn := operation.NewCommand
-	if cursorCommand {
-		createCmdFn = operation.NewCursorCommand
-	}
-	return createCmdFn(runCmdDoc).
-		Session(sess).CommandMonitor(db.client.monitor).
+	var op *operation.Command
+	switch cursorCommand {
```

Comment:
Any reason to `switch` on this bool instead of if + else?

Reply:
Just personal preference. I've usually seen people avoid if/else chains in Go and prefer `switch` instead.